### ISSUE TITLE
Silk protein length

### ISSUE THREAD

**Issue by dobrosketchkun** [thethoughtemporium/Whose-gene-is-it-anyway#13]
As you said in the video, your protein is much smaller than real ones. Maybe it's not a big deal, but what do you think about the [SpyCatcher-SpyTag](https://en.wikipedia.org/wiki/SpyCatcher) approach? The head-on polymerization approach can result in cyclic proteins (something like [this](https://link.springer.com/protocol/10.1007/978-1-0716-0434-2_2#Fig10)), but there is a modification which allows bypassing this. In [Fierer et. al 2014](https://doi.org/10.1073/pnas.1315776111) authors developed a three-component approach - two tags and one ["SpyLigase"]( https://www.pnas.org/content/111/13/E1176#F3). All the sequences are in [supplementary materials](https://www.pnas.org/content/pnas/suppl/2014/03/14/1315776111.DCSupplemental/pnas.201315776SI.pdf).  They are quite small:

>SpyLigase
MSYYHHHHHHDYDGQSGDGKELAGATMELRDSSGKTISTWISDGQVKDFYLYPGKYTFVETAAPDGYEVATAITFTVNEQGQVTVNGKATKGGSGGSGGSGEDSATHI
>SpyTag
AHIVMVDAYKPTK
>KTag
ATHIKFSKRD

So, basically, my idea is - you add KTag and SpyTag on N and C ends of your silk construction, modify SpyLigase to be produced by another strain of yeast out of cells, as with silk. Then you combine super from SpyLigase strain and silk suspension, and theoretically, it will polymerize (maybe it should be done on an ultrasound bath or something). Or you can grow these two strains in one bioreactor and silk will polymerize as it shows out of a cell.

P.S. 
If there is a paywall, you can use [this](https://scihub.wikicn.top/), _but it's not particularly legal._ Or I can upload those papers by myself in here.

**Comment by the-thought-emporium** [comment]
Wow this looks awesome! I'll need to read up more but I've been looking for something like this and this is way sleeker than the systems I'd found. 

Also, we never shame the use of sci-hub here. I plan to thank Alexandra in the paper we write
